### PR TITLE
checkbox of candidate

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Add buttons(send and add to work)
- Add logics for checkbox of the candidate
- Change size of select and table
- Add export to excel

Снимок экрана 2021-11-14 в 7 47 53 PM

https://trello.com/c/pu6ODxVC/78-us71-checkbox-of-the-candidate
https://trello.com/c/qEqwDgY5/135-fe-add-export-to-excel-on-candidatelist-page

### CODE REVIEW

File: src/components/linkFormatter/LinkFormatter.sass [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.linkFormatterWrapper
+    cursor: pointer
+
+.linkFormatterWrapper:hover 
+    color: #0082ca;
```
Comment (beliavskaja) [inline src/components/linkFormatter/LinkFormatter.sass:5]: please remove ;

Comment (ValeryiaMIRON) [reply src/components/linkFormatter/LinkFormatter.sass]: ok

File: src/pages/candidates/Candidates.jsx [hunk 2]
```diff
@@ -78,8 +74,12 @@ const Candidates = () => {
 
   const newListOfCandidates = reformatCandidates(listOfCandidates);
 
+  const onBtExport = () => {
```
Comment (itsnotpolza) [inline src/pages/candidates/Candidates.jsx:77]: what is the meaning of this name

File: src/pages/home/Home.sass [hunk 3]
```diff
@@ -18,7 +18,7 @@
 .card
   margin: 10px
   padding: 0px 0px 20px 0px
-  height: 1300px
+  height: 100vh
```
Comment (itsnotpolza) [inline src/pages/home/Home.sass:21]: this change is not in the scope of the task

File: src/pages/candidates/Candidates.jsx (hunk 2 above)
Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

File: src/pages/home/Home.sass (hunk 3 above)
Comment (ValeryiaMIRON) [reply src/pages/home/Home.sass]: done

**Review by nsukhotskaya — APPROVED** [review]

**Review by beliavskaja — APPROVED** [review]

**Pull request merged**